crates/host/src/wasmbus/event.rs

```diff
@@ -254,3 +254,17 @@ pub(crate) async fn publish(
         .await
         .with_context(|| format!("failed to publish `{name}` event"))
 }
+
+pub fn labels_changed(
+    host_id: impl AsRef<str>,
+    changed: impl Into<HashMap<String, String>>,
```

Comment:
I think this actually requires a bit of state on the other side for listeners. If I want to do something when a label is added, listeners have to save a little bit of state across the batches of labels that come through for a host (how do I know if a label value for a given host has added, updated, or changed, or was just removed?), to avoid doing some possibly costly operation twice.

With the rename though, `labels_changed` is generic enough to just be a simple list of all the labels over and over. Maybe the other events will have more specific names like `label_updated` or `label_added` or `label_removed` and have a smaller slice of information.

Reply:
Yeah I don't think anything right now needs to know exactly what changed, just that it did change